springdoc-openapi-common/src/main/java/org/springdoc/api/AbstractOpenApiResource.java

```diff
@@ -381,6 +391,9 @@ protected synchronized OpenAPI getOpenApi(Locale locale) {
 	 */
 	protected void calculatePath(HandlerMethod handlerMethod,
 			RouterOperation routerOperation, Locale locale, OpenAPI openAPI) {
+
+		customiseRouterOperation(routerOperation, handlerMethod);
```

Comment:
Isn't it better to receive routerOperation as a return for better understanding?
(Yes. I know the given routerOperation internal property is changed through customiseRouterOperation)
```suggestion
		routerOperation = customiseRouterOperation(routerOperation, handlerMethod);
```

Reply:
@junoyoon looks good, thanks.